srcpkgs/deadbeef/template

```diff
@@ -38,18 +31,22 @@ if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	makedepends+=" musl-legacy-compat"
 fi
 
-post_extract() {
-	mv deadbeef-${version}/* .
-	rm -rf external/apbuild
-	rm -rf external/mp4p
-	mv -v apbuild-${_apbuild_commit} external/apbuild
-	mv -v mp4p-${_mp4p_commit} external/mp4p
+do_fetch() {
+       rm -rf deadbeef-${version}
+       git clone --recurse-submodules https://github.com/DeaDBeeF-Player/deadbeef 
+       mv deadbeef deadbeef-${version}
+       cd deadbeef-${version}
+       git checkout ${_release_commit}
 }
 
 pre_configure() {
 	NOCONFIGURE=1 ./autogen.sh
 }
 
+do_configure() {
+        CC=clang CXX=clang++ ./configure ${configure_args}
```

Comment:
This break cross compilation.

Reply:
the new version of deadbeef is only built with clang 
https://github.com/DeaDBeeF-Player/deadbeef/blob/c8e39422eaa72927dee6cb4bafa7208584142f2d/README#L14